### DISCUSSION TITLE
New to MeshCentral

### DISCUSSION THREAD

**Discussion by macooper59** [Ylianst/MeshCentral#6603] [General]
Hey Guys,
I am using Docker Version: 27.3.1 on top of Ubuntu Server 22.04 completely updated. I just created a docker-compose file to deploy MeshCentral to try it out and after creating a user account and trying to login, I get the following error:
**_Invalid origin in HTTP request, click to reconnect._**
![MeshCentralError](https://github.com/user-attachments/assets/f4c2c3e1-7197-45ef-91e2-53243af597f8)

I attached the screenshot for good measure. 
Please let me know if there as anything I can do to fix this. Thank you for your time.
Michael

**Comment by si458** [comment]
` Invalid origin in HTTP request` is basically a security mechanism to prevent bad actors
you should be accessing the url using whatever dns name you have set inside of `cert` which is located inside of `config.json` inside of `meshcentral-data`
chances are `cert` is set to `localhost` if i remember with docker
so you need to edit your `config.json` file and change `cert` to a DNS name which gets resolved to your meshcentral server ip 
then restart meshcentral container 👍

**Reply by macooper59** [reply]
Thank you sir.
